src/test/fuzz/net.cpp

```diff
@@ -77,3 +77,33 @@ FUZZ_TARGET(net, .init = initialize_net)
     (void)node.HasPermission(net_permission_flags);
     (void)node.ConnectedThroughNetwork();
 }
+
+FUZZ_TARGET(local_address, .init = initialize_net)
+{
+    FuzzedDataProvider fuzzed_data_provider(buffer.data(), buffer.size());
+    SetMockTime(ConsumeTime(fuzzed_data_provider));
+    CService service{ConsumeService(fuzzed_data_provider)};
+    LIMITED_WHILE(fuzzed_data_provider.ConsumeBool(), 10000) {
+        CallOneOf(
+            fuzzed_data_provider,
+            [&] {
+                service = ConsumeService(fuzzed_data_provider);
+            },
+            [&] {
+                const bool added{AddLocal(service, fuzzed_data_provider.ConsumeIntegralInRange<int>(0, 5))};
```

Comment:
This harness will be non-deterministic and run out of memory since local addresses are stored in a global map. You'll need to reset [`mapLocalHost`](https://github.com/bitcoin/bitcoin/blob/ba907f96ad37c09c49c0e1532fad118fcb8dd4a8/src/net.cpp#L118) each iteration.

Reply:
Done.